internal/servers/controller/handlers/authmethods/authmethod_service.go

```diff
@@ -734,10 +734,12 @@ func validateCreateRequest(req *pbs.CreateAuthMethodRequest) error {
 						}
 					}
 				}
-				if attrs.GetApiUrlPrefix() != nil {
+				if strings.TrimSpace(attrs.GetApiUrlPrefix().GetValue()) == "" {
+					// TODO: When we start accepting the address used in the request make this an optional field.
```

Comment:
ty for the TODO

Reply:
Your welcome.  Do you think we need to have this check in more places than just the API level?